cmd/dlt/accountroles/cmd.go

```diff
@@ -227,7 +230,21 @@ func deleteAccountRoles(r *rosa.Runtime, env string, prefix string, clusters []*
 		if err != nil {
 			return fmt.Errorf("There was an error getting the policy: %v", err)
 		}
-		commands := buildCommand(finalRoleList, policyMap, arbitraryPolicyMap, managedPolicies)
+
+		// Get HCP shared vpc policy details if the user is deleting roles related to HCP shared vpc
+		policiesOutput := make([]*iam.GetPolicyOutput, 0)
+		if roles.CheckIfRolesAreHcpSharedVpc(r, finalRoleList) &&
+			confirm.Prompt(true, "Attempt to delete Hosted CP shared VPC policies?") {
```

Comment:
Would "attempt to delete" be the proper wording for this?
This would just be building the commands for manual mode right?

Reply:
Ah yeah, I think that's a good point. Will change to something along the lines of `Create command to delete ...` - no need for `attempt` here I think anyway